ci/benchmark/build.sh

```diff
@@ -0,0 +1,210 @@
+# BUILD - Build libcudf, cuDF and dask_cudf from source
+################################################################################
+
+logger "Build libcudf..."
+if [[ ${BUILD_MODE} == "pull-request" ]]; then
+    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests --ptds
+else
+    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests -l --ptds
+fi
+
+################################################################################
+# BENCHMARK - Run and parse libcudf and cuDF benchmarks
+################################################################################
+
+logger "Running benchmarks..."
+
+#Download GBench results Parser
+curl -L https://raw.githubusercontent.com/rapidsai/benchmark/main/parser/GBenchToASV.py --output GBenchToASV.py
+
+###
+# Generate Metadata for dependencies
+###
+
+# Concatenate dependency arrays, convert to JSON array,
+# and remove duplicates.
+X=("${CUDF_DEPS[@]}" "${LIBCUDF_DEPS[@]}")
+DEPS=$(printf '%s\n' "${X[@]}" | jq -R . | jq -s 'unique')
+
+# Build object with k/v pairs of "dependency:version"
+DEP_VER_DICT=$(jq -n '{}')
+for DEP in $(echo "${DEPS}" | jq -r '.[]'); do
+  VER=$(conda list | grep "^${DEP}" | awk '{print $2"-"$3}')
+  DEP_VER_DICT=$(echo "${DEP_VER_DICT}" | jq -c --arg DEP "${DEP}" --arg VER "${VER}" '. + { ($DEP): $VER }')
+done
+
+# Pass in an array of dependencies to get a dict of "dependency:version"
+function getReqs() {
+  local DEPS_ARR=("$@")
+  local REQS="{}"
+  for DEP in "${DEPS_ARR[@]}"; do
+    VER=$(echo "${DEP_VER_DICT}" | jq -r --arg DEP "${DEP}" '.[$DEP]')
+    REQS=$(echo "${REQS}" | jq -c --arg DEP "${DEP}" --arg VER "${VER}" '. + { ($DEP): $VER }')
+  done
+
+  echo "${REQS}"
+}
+
+###
+# Run LIBCUDF Benchmarks
+###
+
+REQS=$(getReqs "${LIBCUDF_DEPS[@]}")
+
+BENCHMARK_META=$(jq -n \
+  --arg NODE "${NODE_NAME}" \
+  --arg BRANCH "branch-${MINOR_VERSION}" \
+  --argjson REQS "${REQS}" '
+  {
+    "machineName": $NODE,
+    "commitBranch": $BRANCH,
+    "requirements": $REQS
+  }
+')
+
+echo "Benchmark meta:"
+echo "${BENCHMARK_META}" | jq "."
+
+mkdir -p ${WORKSPACE}/tmp/benchmark
+touch ${WORKSPACE}/tmp/benchmark/benchmarks.txt
+ls ${GBENCH_BENCHMARKS_DIR} > ${WORKSPACE}/tmp/benchmark/benchmarks.txt
+
+#Disable error aborting while tests run, failed tests will not generate data
+logger "Running libcudf GBenchmarks..."
+cd ${GBENCH_BENCHMARKS_DIR}
+set +e
+while read BENCH;
+do
+    nvidia-smi
+    ./${BENCH} --benchmark_out=${BENCH}.json --benchmark_out_format=json
+    EXITCODE=$?
+    if [[ ${EXITCODE} != 0 ]]; then
+        rm ./${BENCH}.json
+	JOBEXITCODE=1
+    fi
+done < ${WORKSPACE}/tmp/benchmark/benchmarks.txt
+set -e
+
+rm ${WORKSPACE}/tmp/benchmark/benchmarks.txt
+cd ${WORKSPACE}
+mv ${GBENCH_BENCHMARKS_DIR}/*.json ${WORKSPACE}/tmp/benchmark/
+python GBenchToASV.py -d  ${WORKSPACE}/tmp/benchmark/ -t ${ASVRESULTS_DIR} -n libcudf -b branch-${MINOR_VERSION} -r ${BENCHMARK_META} 
```

Comment:
only the requirements should be passed to the parser here. so use `-r "${REQS}"` instead of `-r "${BENCHMARK_META}"`. (make sure to include the enclosing quotations `" "`)